t/api/10-jobgroups.t

```diff
@@ -125,15 +127,58 @@ subtest 'create' => sub() {
                 'size_limit_gb'                  => 200,
                 'id'                             => $new_id
             },
-        ]);
+        ],
+        'list created job group'
+    );
+};
+
+subtest 'update job group' => sub() {
+    my $new_id = $t->post_ok(
+        '/api/v1/parent_groups',
+        form => {
+            name                                   => 'Update parent',
+            default_keep_important_logs_in_days    => 100,
+            default_keep_important_results_in_days => 366
+        })->tx->res->json->{id};
+
+    my $put = $t->put_ok(
+        '/api/v1/job_groups/1001',
+        form => {
+            size_limit_gb               => 101,
+            description                 => 'Test',
+            keep_important_logs_in_days => 45,
+            parent_id                   => $new_id,
+        });
+
+    my $get = $t->get_ok('/api/v1/job_groups/1001')->status_is(200);
+    is_deeply(
+        $get->tx->res->json,
+        [
+            {
+                'name'                           => 'opensuse',
```

Comment:
as you have it 8 lines further above without the quotes in hashes, please unify this. Even if we don't enforce perlcritic for t/*